### DISCUSSION TITLE
How to display selected day/dates (when DateRangePicker)?

### DISCUSSION THREAD

**Discussion by filippos13** [wojtekmaj/react-calendar#579] [Q&A] [answered]
Hi! I'm using react-calendar and I find it extremely good and fast. I would like to know how to display selected day/dates (when DateRangePicker)?

**Comment by wojtekmaj** [comment, marked as answer]
Since depending on the configuration React-Calendar may return a date or an array of dates, here's what I'm using in test suite to render currently selected dates:

https://github.com/wojtekmaj/react-calendar/blob/41b1981b8e3ed33bc6f53a207e710e5709fbfeb9/test/Test.jsx#L115-L136

Obviously, you'll need to have access to `value`, so you will need to use `onChange` to get the new value whenever it's changed.

**Reply by filippos13** [reply]
Thanks! I got an idea how to implement it! Although, I face a weird situation. I click on a date and the value it returns me is one day earlier (I talk about the start date. The end date is correct). Correct me if I am wrong (it's the first time I work with dates & calendars).
![date](https://user-images.githubusercontent.com/4280123/149827005-f2d7b1d7-8dd9-42ce-a287-e5e45c3b529f.JPG)

This is the code of the calendar. I display it so you know I didn't change anything to the logic of the dates.
![calendar](https://user-images.githubusercontent.com/4280123/149827382-579cc84c-4870-4540-a6d6-db7c40f772fa.JPG)

**Reply by wojtekmaj** [reply]
What you're seeing in the console is just ISO representation of the date. Meaning it's being represented in UTC+0 timezone instead of your own, resulting in this offset. In other words, simplifying a lot, when it's Tuesday midnight at your place, it's Monday 21:00 in London.

**Comment by filippos13** [comment]
Thanks for everything! 👍
